tests/python/driver/tvmc/test_autotuner.py

```diff
@@ -182,3 +182,18 @@ def test_tune_rpc_tracker_parsing(mock_load_model, mock_tune_model, mock_auto_sc
     assert "10.0.0.1" == kwargs["hostname"]
     assert "port" in kwargs
     assert 9999 == kwargs["port"]
+
```

Comment:
possible to add a test using e.g. pyplot svg backend and a golden svg file?

Reply:
I will look into this.